cypress/e2e/19-execution.cy.ts

```diff
@@ -214,6 +214,33 @@ describe('Execution', () => {
 		workflowPage.getters.clearExecutionDataButton().should('not.exist');
 	});
 
+	it('should test workflow with specific trigger node', () => {
+		cy.createFixtureWorkflow('Manual_webhook_wait.json');
+		cy.intercept('POST', '/rest/workflows/**/run?**').as('workflowRun');
+
+		workflowPage.getters.zoomToFitButton().click();
```

Comment:
we are moving away from these classes. Could you use rerewrite this using functions instead? 

You can see how we use that here
https://github.com/n8n-io/n8n/blob/master/cypress/composables/workflow.ts